installer/release.sh

```diff
@@ -76,7 +30,7 @@ do
 		skip_files=t
 		;;
 	-r|--render-release-notes)
-		render_release_notes &&
+		sh ../render-release-notes.sh
```

Comment:
`sh` is unnecessary, but I would really like `--outputdir=.` and `--csspath=usr/share/git/` to be passed to the script (no need for that script to know where the InnoSetup config wants the files to be).